<?xml version="1.0" encoding="utf-8"?>
<test-run id="2" name="ConsoleApp1.exe" fullname="C:/Dev/MyClub/src/ConsoleApp1/bin/Debug/net461/win7-x64/ConsoleApp1.exe" testcasecount="2" result="Failed" start-time="2016-05-22 09:59:04Z" end-time="2016-05-22 09:59:10Z" duration="5.405019" total="2" passed="1" failed="1" inconclusive="0" skipped="0" asserts="2" random-seed="1609902300">
  <command-line><![CDATA["C:\Dev\MyClub\src\ConsoleApp1\bin\Debug\net461\win7-x64\ConsoleApp1.exe" ]]></command-line>
  <filter />
  <test-suite type="Assembly" id="1003" name="ConsoleApp1.exe" fullname="C:/Dev/MyClub/src/ConsoleApp1/bin/Debug/net461/win7-x64/ConsoleApp1.exe" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2016-05-22 09:59:04Z" end-time="2016-05-22 09:59:10Z" duration="5.405019" total="2" passed="1" failed="1" inconclusive="0" skipped="0" asserts="2">
    <environment framework-version="3.2.1.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 6.2.9200.0" platform="Win32NT" cwd="C:\Dev\MyClub\src\ConsoleApp1" machine-name="DESKTOP-CG7810V" user="gvd8" user-domain="DESKTOP-CG7810V" culture="en-US" uiculture="en-US" os-architecture="x64" />
    <settings>
      <setting name="WorkDirectory" value="C:\Dev\MyClub\src\ConsoleApp1" />
      <setting name="NumberOfTestWorkers" value="4" />
    </settings>
    <properties>
      <property name="_PID" value="6900" />
      <property name="_APPDOMAIN" value="ConsoleApp1.exe" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
    </failure>
    <test-suite type="TestFixture" id="1000" name="SampleTest" fullname="SampleTest" classname="SampleTest" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2016-05-22 09:59:04Z" end-time="2016-05-22 09:59:10Z" duration="5.384542" total="2" passed="1" failed="1" inconclusive="0" skipped="0" asserts="2">
      <failure>
        <message><![CDATA[One or more child tests had errors]]></message>
      </failure>
      <test-case id="1002" name="FailingTest" fullname="SampleTest.FailingTest" methodname="FailingTest" classname="SampleTest" runstate="Runnable" seed="1630670862" result="Failed" start-time="2016-05-22 09:59:04Z" end-time="2016-05-22 09:59:10Z" duration="5.366534" asserts="1">
        <failure>
          <message><![CDATA[  Expected: 5
  But was:  4
]]></message>
          <stack-trace><![CDATA[at SampleTest.FailingTest() in C:\Dev\MyClub\src\ConsoleApp1\SampleTest.cs:line 15
]]></stack-trace>
        </failure>
      </test-case>
      <test-case id="1001" name="PassingTest" fullname="SampleTest.PassingTest" methodname="PassingTest" classname="SampleTest" runstate="Runnable" seed="798259755" result="Passed" start-time="2016-05-22 09:59:10Z" end-time="2016-05-22 09:59:10Z" duration="0.000622" asserts="1" />
    </test-suite>
  </test-suite>
</test-run>